src/OrleansCodeGenerator/CodeGeneratorCommon.cs

```diff
@@ -108,7 +109,10 @@ public static GeneratedAssembly CompileAssembly(GeneratedSyntax generatedSyntax,
             var symbolStream = emitDebugSymbols ? new MemoryStream() : null;
             try
             {
-                var compilationResult = compilation.Emit(outputStream, symbolStream);
+                var emitOptions = new EmitOptions()
+                    .WithDebugInformationFormat(DebugInformationFormat.PortablePdb);
+
+                var compilationResult = compilation.Emit(outputStream, symbolStream, options: emitOptions);
```

Comment:
would this work well even if `emitDebugSymbols` is false (and hence `symbolStream` is null)?